### PR TITLE
Add MANIFEST_NAME property

### PR DESCRIPTION
**Pull request by glevand**
So a manifest file can be specified.

### CODE REVIEW

File: os/manifest.groovy [hunk 1]
```diff
@@ -162,7 +165,7 @@ then
 fi
 
 MANIFEST_BRANCH="${GIT_BRANCH##*/}"
-MANIFEST_NAME="${MANIFEST_BRANCH}.xml"
+MANIFEST_NAME="${MANIFEST_NAME}.xml"
```
Comment (dm0-) [inline os/manifest.groovy]: This will need to be set in the environment to be used here.

Comment (glevand) [reply os/manifest.groovy]: The parameter variables should be available as shell environment variables, and seems to work in my tests.  See: https://wiki.jenkins.io/display/JENKINS/Parameterized+Build

**Comment by glevand** [comment]
The current behavior is that if a MANIFEST_BRANCH is specified, then the manifest file name needs to match it in the repository.  So for example if the branch name is `test1` then the manifest file must be `test1.xml`.  With this change the default manifest file is `master.xml' for any MANIFEST_BRANCH specified.  I thought this would be more natural usage.

Did you want the MANIFEST_NAME to match MANIFEST_BRANCH?

**Comment by dm0-** [comment]
Our builds currently require the branch to match the manifest name used.  (The `build-*` branches are for development of stable/beta/alpha-minor releases.)  An alternative is to point it at the `default.xml` symlink, which should be a reliable indicator of the branch name now that it is an automated part of our release tagging process.

**Comment by glevand** [comment]
Changed `defaultValue: 'master'` to `defaultValue: 'default'`.  This seems it will work if your branches have the symlink set properly.

**Pull request merged**